### PR TITLE
CSS suppressed for eater card hover effect on small screen

### PR DESCRIPTION
**Pull request by richsilv**
Event handler added to show buttons when the profile image is clicked, which will only be possible on small screens (as other screens will have the image occluded on hover).

### CODE REVIEW

File: client/makelunch.js [hunk 1]
```diff
@@ -130,6 +130,10 @@ Template.registerHelper('todaysISODate', function () {
 })
 
 Template.card.events({
+  'click .photoFrame': function (evt, tpl) {
+    $('.card').removeClass('visible')
+    tpl.$('.card').addClass('visible')
```
Comment (liamwooding) [inline client/makelunch.js:135]: Why the different styles of selector?

File: client/makelunch.less [hunk 2]
```diff
@@ -175,7 +175,18 @@ body.home .card {
       .btn { display: block; }
     }
   }
-  &:hover, &:focus {
+  @media (min-width: 768px) {
```
Comment (liamwooding) [inline client/makelunch.less]: Although we (I) did inline media queries a lot on School of Wok, I think we've all decided it's not great in the long run - have them all together in a block at the bottom of the file.

File: client/makelunch.js (hunk 1 above)
Comment (richsilv) [inline client/makelunch.js:135]: The first one removes it everywhere, the second one adds it only to the template you've clicked.

File: client/makelunch.less (hunk 2 above)
Comment (richsilv) [inline client/makelunch.less]: Changed as requested.  We should really refactor the LESS into different files as it's pretty hard to find your way around at present.

File: client/makelunch.less [hunk 3]
```diff
@@ -361,6 +361,15 @@ body.editperson .editimg img {
       > nav { max-height: auto; }
     }
   }
+  .card:hover {
```
Comment (liamwooding) [inline client/makelunch.less]: There's already a `body.home card` block a few lines above, this should go in there

Comment (richsilv) [inline client/makelunch.less]: This is why we need to refactor the LESS!

Comment (richsilv) [inline client/makelunch.less]: Sorted.

**Pull request merged**